src/Workspaces/VisualBasic/Portable/Recommendations/VisualBasicRecommendationService.vb

```diff
@@ -138,24 +138,23 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Recommendations
             ' "Func(Of" tends to get in the way of typing "Function". Exclude System.Func from expression
             ' contexts, except within GetType
             If Not context.TargetToken.IsKind(SyntaxKind.OpenParenToken) OrElse
-               Not context.TargetToken.Parent.IsKind(SyntaxKind.GetTypeExpression) Then
+                    Not context.TargetToken.Parent.IsKind(SyntaxKind.GetTypeExpression) Then
 
-                symbols = symbols.Where(Function(s) Not IsSystemFunc(s))
+                symbols = symbols.Where(Function(s) Not IsInEligibleDelegate(s))
```

Comment:
This should probably be "Ineligible" as one word, since I think that's what you actually meant.